OpenXmlPowerTools/MetricsGetter.cs

```diff
@@ -40,16 +40,11 @@ public class MetricsGetterSettings
 
     public class MetricsGetter
     {
-        private static Lazy<Graphics> Graphics {
-            get
-            {
-                return new Lazy<Graphics>(() =>
-                {
-                    Image image = new Bitmap(1, 1);
-                    return System.Drawing.Graphics.FromImage(image);
-                });
-            }
-        }
+        private static Lazy<Graphics> Graphics { get; } = new Lazy<Graphics>(() =>
+        {
+            Image image = new Bitmap(1, 1);
```

Comment:
Just wondering now, how expensive is the graphics invocation? Does it need to be lazy? I'm fine with how it is, but just wondered if any perf testing was done to see if this was helpful?

Reply:
No I didn't check, but instinctively this kind of trick takes a lot of time and memory if repeated a lot.